pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -686,7 +687,7 @@ def _check_and_init_precision(self) -> PrecisionPlugin:
 
                 if isinstance(self.strategy, (DDPShardedStrategy, DDPSpawnShardedStrategy)):
                     return ShardedNativeMixedPrecisionPlugin(self._precision_flag, device)
-                if isinstance(self.strategy, DDPFullyShardedStrategy):
+                if isinstance(self.strategy, (DDPFullyShardedStrategy, DDPFullyShardedNativeStrategy)):
```

Comment:
clip_grad_by_norm was supported in the PyTorch nightly, not PyTorch 1.11:(. mix precision will be landed soon (mostly this week), and will be available in PyTorch nightly as well...

Reply:
@zhaojuanmao this is good to know :) do you know if offloading is also compatible with mixed precision?

Reply:
@SeanNaren mixed precision was just landed this week to nightly. Yes, mixed precision can be used with or without cpu offloading. Also it will be more flexible now to specify computation parameter types or buffer types or communication/reduce types, these types are not limited to fp16. bfloat 16 or other types should be supported as well.